examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js

```diff
@@ -144,6 +142,13 @@ export default class MeshLayer extends SimpleMeshLayer {
       });
     }
 
+    if (segmentationMode && segmentationData) {
+      model.setUniforms({
+        // eslint-disable-next-line camelcase
+        u_pickSegmentation: true
```

Comment:
The way this is coded, it cannot be switched back if the props change?

```suggestion
      const pickFeatures = segmentationMode && segmentationData;
      model.setUniforms({
        // eslint-disable-next-line camelcase
        u_pickSegmentation: pickFeatures
```

Reply:
Fixed